scripts.js

```diff
@@ -6,21 +6,68 @@ let oneDay = 1000*60*60*24;
 let daysLeft = Math.round((endOfChallenge - currentDate)/oneDay);
 let daysLeftDisplay = document.querySelector('#daysLeftDisplay');
 let bookFrequencyDisplay = document.querySelector('#bookFrequencyDisplay');
-console.log(currentDate);
+let frequencyDisplayText = document.querySelector('#frequencyDisplayText');
+
+const writeToLocalStorage = (goal, read) => {
```

Comment:
This function writes to the localstorage, when the goal value and the read value are passed in. (This is only written if there is a Storage functionality in the browser)